templates/service-web.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: web
+spec:
+  type: NodePort
```

Comment:
Have looked at keystone and horizon charts and tried to do something similar

It seems that NodePorts are set to True by mumoshu